tests/test_seasonal_water_yield_regression.py

```diff
@@ -782,7 +783,7 @@ def _assert_regression_results_equal(
             result_vector_path (string): path to the summary shapefile
                 produced by the SWY model.
             agg_results_path (string): path to a csv file that has the
-                expected aggregated_results.shp table in the form of
+                expected aggregated_results_swy.shp table in the form of

```

Comment:
Come to think of it, this, too, would ideally use the module-level variable somehow if it needed to refer to the filename.